ext/odbc/php_odbc.c

```diff
@@ -2089,9 +2079,7 @@ PHP_FUNCTION(odbc_free_result)
 	result = Z_ODBC_RESULT_P(pv_res);
 	CHECK_ODBC_RESULT(result);
 
-	if (result->conn_ptr) {
```

Comment:
we can call `odbc_result_free()` unconditionally, since `CHECK_ODBC_RESULT` already checks the same